WELD.md

```diff
@@ -139,3 +104,36 @@ python weld.py
 ### Goal:
 
 1.  Track the submodule (recipe) version with the current data repository.
+
+## Step 5: Commit most weld results to Github
+
+### Procedure:
+Depending on the size of your dataset, it may not be practical to commit all weld results to Github. Our team has decided to commit all summary figures and results but not the per-site paint or spots data to Github. You can tailor the following commit code to suit your data storage preferences.
```

Comment:
```suggestion
Depending on the size of your dataset, it may not be practical to commit all weld results to Github.
Our team has decided to commit all summary figures and results but not the per-site paint or spots data to Github.
You can tailor the following commit code to suit your data storage preferences.
```

One-sentence-per-line makes future edits easier.

Please make this change throughout the document.